src/content/app/genome-browser/components/browser-nav-icon/BrowserNavIcon.tsx

```diff
@@ -23,7 +23,7 @@ import ImageButton from 'src/shared/components/image-button/ImageButton';
 
 import { browserNavIconActionMap } from 'src/content/app/genome-browser/state/browser-nav/browserNavSlice';
 
-import { BrowserNavItem } from 'src/content/app/genome-browser/components/browser-nav/browserNavConfig';
+import { BrowserNavItem } from 'src/content/app/genome-browser/components/browser-nav-icon/browserNavConfig';
```

Comment:
What does the component called `browser-nav-icon` do? Is it now responsible for the five buttons that move the genome browser left/right, zoom in/out and refocus on the focus object? If yes, then should we rename the directory (`browser-nav-buttons`, or something)?

Reply:
Agree with this. `BrowserNavIcon` uses `ImageButton` which itself suggests that it is more than a simple icon. I have renamed the directory and the component to be `BrowserNavButton`.